benefits/core/models.py

```diff
@@ -459,6 +460,12 @@ def claims_scheme(self):
             return self.claims_provider.scheme
         return self.claims_scheme_override
 
+    @property
+    def claims_all_claims(self):
+        claims = [self.claims_eligibility_claim]
+        claims.extend(self.claims_extra_claims.split())
```

Comment:
I like this helper to calculate the full list in the model.

However this call to `.split()` will throw an error if `self.claims_extra_claims` is `None` (the default).

We should either set the default to an empty string, or handle when this is `None`.

Reply:
I agree and want to add that we should have a unit test that covers this.

Reply:
Ah, thanks both for pointing this out 👍
In f9eaa59d68b1dfb9eccb791bfc0bfe018095fde3 we now have a handler to make sure we don't get an error and a test for when we have `self.claims_extra_claims` set to `None`, one claim, or two claims separated by a space.